content/development/getting-started.html

```diff
@@ -5,9 +5,12 @@
 tags = []
 +++
 
-The information on this page is intended to explain how you can get started with Haiku development.
+<p>Developers are the driving force behind Haiku, and without them, we would not be where we are today. There's always a lot of work to be done, so this is by far one of the most important jobs anyone in the Haiku community can take up on. Consequently, it will probably require a large investment of your time. Even so, it’s hard to put into words the pleasure of seeing something you’ve built help many other people. Newcomers will need to know or be willing to learn C++, as well as the BeOS/Haiku API.
+
+The information on this page is intended to explain how you can get started with Haiku development.</p>
```

Comment:
I wouldn't have guessed from the page title...

Reply:
Removed?